remotespark/livyclientlib/exceptions.py

```diff
@@ -0,0 +1,34 @@
+class LivyClientLibException(Exception):
```

Comment:
Probably not, I don't think directly raising an exception that is not a `LivyClientLibException` is a good idea.  We can document it.